test/specs/cli.js

```diff
@@ -1,9 +1,5 @@
-import test from 'ava';
-import sinon from 'sinon';
```

Comment:
With the globals we have now, you don't need to import testing rig modules.  Just the stuff you are testing.